requirements.dev.txt

```diff
@@ -2,7 +2,11 @@ numpy
 apache-beam
 absl-py
 pyspark
+
+# pytest stuff
```

Comment:
do we need this line?

Reply:
It added for readability for separating different requiremnts. Not a bad idea.

Reply:
Thanks, SG